picard/ui/mainwindow.py

```diff
@@ -422,6 +422,24 @@ def create_actions(self):
         self.cd_lookup_action.setShortcut(QtGui.QKeySequence(_("Ctrl+K")))
         self.cd_lookup_action.triggered.connect(self.tagger.lookup_cd)
 
+        self.cd_lookup_menu = QtWidgets.QMenu(_("Lookup &CD..."))
+        self.cd_lookup_menu.triggered.connect(self.tagger.lookup_cd)
+        self.cd_lookup_action.setEnabled(False)
+        if discid is None:
+            log.warning("CDROM: discid library not found - Lookup CD functionality disabled")
+        else:
+            drives = get_cdrom_drives()
+            if len(drives) == 0:
+                log.warning("CDROM: No CD-ROM drives found - Lookup CD functionality disabled")
+            else:
+                shortcut_drive = config.setting["cd_lookup_device"].split(",")[0] if len(drives) > 1 else ""
+                self.cd_lookup_action.setEnabled(True)
+                for drive in drives:
+                    action = self.cd_lookup_menu.addAction(drive)
+                    if drive == shortcut_drive:
+                        action.setShortcut(QtGui.QKeySequence(_("Ctrl+K")))
```

Comment:
The same shortcut is used at line 422:
```python
self.cd_lookup_action.setShortcut(QtGui.QKeySequence(_("Ctrl+K")))
```
I think it would be more obvious to have only one place for this shortcut, instead of defining then undefining

Edit: understood, but a bit late